projects/dsp-ui/src/lib/viewer/views/list-view/list-view.component.ts

```diff
@@ -77,6 +79,11 @@ export class ListViewComponent implements OnInit {
         this._doSearch();
     }
 
+    ngOnChanges(): void {
+        this.resources = undefined;
+        this.ngOnInit();
```

Comment:
question: is it a good practice to call ngOnInit inside ngOnChanges?

Reply:
Done in f0268ac